src/main/kotlin/dev/haarolean/gnutardbot/abilities/CheckLinksAbility.kt

```diff
@@ -36,7 +36,8 @@ class CheckLinksAbility(private val bot: TardBot) : AbilityProvider {
         // https://github.com/rubenlagus/TelegramBots/issues/1296
         val update = ctx.update()
         if (!AbilityUtils.isUserMessage(update)) return true
-        return if (Flag.MESSAGE.test(update) || Flag.EDITED_MESSAGE.test(update)) false else false
+        if (Flag.MESSAGE.test(update) || Flag.EDITED_MESSAGE.test(update)) return false
```

Comment:
This whole method looks quite suspicious:
- `isUserMessage(update)` check seems redundant all as we test for the `MESSAGE` and `EDITED_MESSAGE` flags next
- flag test condition looks invalid (should be rather `return true`?) as we're only concerned about new and edited text messages (which possibly contain links).

May I suggest to look closer into this? Thanks